packages/web/src/components/basic/DataController.js

```diff
@@ -56,7 +56,8 @@ class DataController extends Component {
 		let query = DataController.defaultQuery(defaultSelected, props);
 		let customQueryOptions;
 		if (customQuery) {
-			({ query } = customQuery(defaultSelected, props));
+			const customQueryObject = customQuery(defaultSelected, props);
+			query = customQueryObject && customQueryObject.query;
```

Comment:
Can't we just do
`({ query } = customQuery(defaultSelected, props) || {})`

Reply:
Agree, that's what I'm doing in ReactiveComponent.

Reply:
Sure will update this in files